dora/minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -149,8 +162,44 @@ protected void startProxy() throws Exception {
     }
   }
 
+  @Override
+  protected void waitForMasterServing() throws TimeoutException, InterruptedException {
+    CommonUtils.waitFor("master starts serving RPCs", () -> {
+      try (BlockMasterClient blockMasterClient = BlockMasterClient.Factory.create(
+          // There is not a local config in this class, all Properties are set to global.
+          // Properties are only set in the initConfiguration() method,
+          // without any @Rule to set Properties in this class.
```

Comment:
nit:
```suggestion
          // This cluster uses the global configuration singleton instead of a local one
          // Config properties are only set in the initConfiguration() method
```